module-apps/apps-common/popups/Popups.hpp

```diff
@@ -29,6 +29,7 @@ namespace gui
             PowerOff,
             Reboot,
             BedtimeNotification,
+            DeadBatteryWindow,
```

Comment:
I do agree that its confusing. Adding that to our "technical improvements" site next to AppWindow and build system splits :)

Reply:
is it that hard to split though? I believe this would mean right now:
1. moving enum to separate file
2. have separate file for bell and separate for pure 
job done ;)

Reply:
I believe taking such an approach would result in both files having most of their contents identical. I wonder if using the enum class is applicable in this scenario. Maybe it would be better to use templated struct for storing popups types. This way product-specific popups types could be injected at compile time. What do you think @pholat?

Reply:
@swichu91 actually this enum doesn't make too much sense in grand scheme of things :) We most probably should use the manifest template to state what popups are in the selected application. It's meant to have such information. Saying that - it's a major update to the code. At the moment I would be happy with just splitting these.